### PR TITLE
Additional TF 0.13 Fixes

### PR DESCRIPTION
**Pull request by osterman**
## what
* upgrade chatops
* add automatic releasing
* upgrade to go modules
* update any version pinning
* (locally verififed) no `terraform init` errors using cli

## why
* Support Terraform 0.13

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -9,7 +9,7 @@ module "default_label" {
 }
 
 module "notify_slack" {
-  source               = "git::https://github.com/terraform-aws-modules/terraform-aws-notify-slack?ref=tags/v2.3.0"
+  source               = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
```
Comment (Gowiem) [inline main.tf:12]: @osterman Looks like something went wrong here.

Comment (osterman) [reply main.tf:12]: that's a wierd one! =) thanks for catching that.

Comment (osterman) [reply main.tf:12]: wonder if it's because it's an external module `terraform-aws-modules/terraform-aws-notify-slack`

File: test/src/Makefile [hunk 2]
```diff
@@ -1,50 +1,30 @@
...
+docker/test:
+	docker run --name terratest --rm -it -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e GITHUB_TOKEN \
+		-e PATH="/usr/local/terraform/$(TERRAFORM_VERSION)/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" \
+		-v $(CURDIR)/../../:/module/ cloudposse/test-harness:latest -C /module/test/src test
 
 .PHONY : clean
 ## Clean up files
 clean:
-	rm -rf .gopath/ vendor/ $(TF_DATA_DIR)
+	rm -rf ../../examples/complete/*.tfstate*
```
Comment (jamengual) [inline test/src/Makefile:30]: there is no `examples` directory

**Comment by latacora-tomekr** [comment]
Hi all, is this PR currently blocked on @osterman making the changes requested by the review?

**Comment by jamengual** [comment]
> Hi all, is this PR currently blocked on @osterman making the changes requested by the review?

We do not merge PR that have no tests and this is why this is failing

**Review by aknysh — APPROVED** [review]

**Pull request merged**